Automattic/wp-calypso
# Seller Experience - Themes: Theme info popup

#### Proposed Changes

Implement the theme info popup on the seller design picker

Clicking the header info button displays this popup which contains details about the selected theme.

https://user-images.githubusercontent.com/917632/176275792-e7f1ac13-0818-4754-b4eb-68d7c937692e.mp4


#### Testing Instructions

- Go to http://calypso.localhost:3000/setup/designSetup?siteSlug=[slug]&flags=signup/theme-preview-screen,signup/seller-upgrade-modal
- Click the <img width="215" alt="image" src="https://user-images.githubusercontent.com/917632/176276034-c2583330-8a60-46aa-ab8f-9ede4952af00.png"> in the header.
- The popup show above should appear with information about the current theme.


Related to #64782
